__tests__/localJsonStoreAgent.test.ts

```diff
@@ -108,20 +106,26 @@ const setup = async (options?: IAgentOptions): Promise<boolean> => {
   // and `DataStoreJson` if you want to use all the query capabilities of `DataStoreJson`
   databaseFile = options?.context?.databaseFile || `./tmp/local-database-${Math.random().toPrecision(5)}.json`
 
+  try {
```

Comment:
@mirceanis this is the fix for the test failure. Somehow the test env wouldn't have a `tmp` folder, so creating the file would fail (`fs.open` can't create new directories). no idea why this behavior changed, though. Let me know if you see a better fix

Reply:
I suppose that using `fs.mkdir(path, options)` with `options.recursive: true` might be an option. See [docs](https://nodejs.org/api/fs.html#fspromisesmkdirpath-options). It should ensure that all parent dirs are also created if they don't exist.

Reply:
changed this and it seems to work